mods/d2k/chrome/ingame-observer.yaml

```
@@ -313,7 +313,7 @@ Container@OBSERVER_WIDGETS:
 						Container@ECONOMY_STATS_HEADERS:
 							X: 0
 							Y: 0
-							Width: 640
+							Width: 740
```

```suggestion
							Width: 720
```

you are expanding UI by 80, not 100